worker/src/main/java/org/apache/celeborn/service/deploy/worker/congestcontrol/CongestionController.java

```diff
@@ -40,36 +40,57 @@ public class CongestionController {
   private final WorkerSource workerSource;
 
   private final int sampleTimeWindowSeconds;
-  private final long highWatermark;
-  private final long lowWatermark;
+  private final long diskBufferHighWatermark;
+  private final long diskBufferLowWatermark;
+  private final long userProduceSpeedHighWatermark;
+  private final long userProduceSpeedLowWatermark;
+  private final long workerProduceSpeedHighWatermark;
+  private final long workerProduceSpeedLowWatermark;
   private final long userInactiveTimeMills;
 
   private final AtomicBoolean overHighWatermark = new AtomicBoolean(false);
 
   private final BufferStatusHub consumedBufferStatusHub;
 
+  private final BufferStatusHub producedBufferStatusHub;
+
   private final ConcurrentHashMap<UserIdentifier, UserBufferInfo> userBufferStatuses;
 
   private final ScheduledExecutorService removeUserExecutorService;
 
   private final ScheduledExecutorService checkService;
 
+  private final ConcurrentHashMap<UserIdentifier, UserCongestionControlContext>
+      userCongestionContextMap;
+
   protected CongestionController(
       WorkerSource workerSource,
       int sampleTimeWindowSeconds,
-      long highWatermark,
-      long lowWatermark,
+      long diskBufferHighWatermark,
```

Comment:
Can we introduce ConfigService for Worker to support dynamic change these configuration? like `configService.getSystemConfigFromCache....`